include/quic/btstream.hpp

```diff
@@ -177,6 +187,10 @@ namespace oxen::quic
 
         void register_command(std::string endpoint, std::function<void(message)>);
 
+        const Address& local() const { return conn.local(); }
+
+        const Address& remote() const { return conn.remote(); }
+
```

Comment:
It might be more useful to make a public `connection_interface& Stream::connection()` (and then you can just `s.get_conn().local()` here instead of adding these, which seem more connection-related than stream-related).

(As a side effect that would also allow for doing things like having a command on one stream able to easily open other streams or send datagrams on the connection as a result of the first stream's message.)